openedx/features/course_experience/templates/course_experience/course-updates-fragment.html

```diff
@@ -29,20 +29,26 @@
         </div>
     </header>
     <div class="page-content">
-        % if len(plain_html_updates) > 0:
+        % if plain_html_updates and len(plain_html_updates) > 0:
             ${HTML(plain_html_updates)}
         % else:
             <div class="all-updates">
-                % for index, update in enumerate(updates):
-                <article class="updates-article">
-                    % if not update.get("is_error"):
-                    <div class="date">${update.get("date")}</div>
-                    % endif
-                    <div class="article-content">
-                        ${HTML(update.get("content"))}
-                    </div>
-                </article>
-              % endfor
+                % if updates:
+                    % for index, update in enumerate(updates):
+                    <article class="updates-article">
```

Comment:
Nit: we indent HTML inside for/if clauses to make it easier to see where they end.